docs/source/basic_tutorials/consuming_tgi.md

```diff
@@ -75,6 +75,44 @@ To serve both ChatUI and TGI in same environment, simply add your own endpoints
 
 ![ChatUI](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/chatui_screen.png)
 
+## Gradio
+
+Gradio has a `ChatInterface` class to create neat UIs for chatbots. Let's take a look at how to create a chatbot with streaming mode using TGI and Gradio. Assume you are serving your model on port 8080.
```

Comment:
Given you use `huggingface_hub`, I would add a link to the `consuming_tgi` section on `huggingface_hub` client